src/layouts/core-dev-dot-layout/core-dev-dot-layout.module.css

```diff
@@ -21,8 +21,12 @@
   & button {
     font-family: var(--font-body);
 
-    &:hover {
+    &:not(:disabled) {
```

Comment:
TLDR; very non-blocking random thought, feel free to ignore!

Re: the TODO comment above... would we be open to starting with these styles within the button component?

I'm definitely on the fence here; I can't really imagine a `button` where we wouldn't want these styles... however I think it shouldn't be too much of a tradeoff to repeat these styles in every button-ish thing we build.

Alternately... maybe we split this out to a kind of "normalize"-ish stylesheet? I suspect this might be something the design systems team has an opinion on 🤔

Reply:
I kinda like the idea of "normalize-on-the-fly" - eg if there's some other `<button>`-using components that need to be normalized as well (eg the ones in the sidebar), we'd repeat these styles within that component (and helpers seem like a great way to abstract rather than repeat! 👍).

If we go the more traditional normalize route, I'm really not sure where that stylesheet would live 🤔 i guess in most projects it'd be a global stylesheet; in our case since we have the co-located dot-ios I suppose we'd need something scoped to `core-dev-dot-layout` 🤷‍♂️

Reply:
I really like the idea of using a helper for this! For this cursor thing specifically, we could do something like this in `src/pages/style.css`:

```
.g-button-cursor-pointer {
  &:not(:disabled) {
    cursor: pointer;
  }

  &:disabled {
    cursor: not-allowed;
  }
}
```

I assume we'd want a separate one for handling `font-family`, or maybe that one actually stays here since font probably won't change between buttons? 🤔

Reply:
☝️ yeah i dig that! another approach could be to tie it to the HTML element to make it a bit more generic, and to signal the intent (i think?) to apply this to all `button` elements, eg:

```
/* src/styles/normalize.module.css */
.normalize-button {
  font-family: var(--font-body);
  font-size: 100%;
  line-height: inherit;
  
  &:not(:disabled) {
    cursor: pointer;
  }
 
  &:disabled {
    cursor: not-allowed;
  }
}

/* src/components/some-button/some-button.module.css */
.root {
  composes: normalize-button from "src/styles/normalize.module.css";
  /* ... other implementation-specific styles */
}
```

personally kind of on the fence... a "standard" global `normalize.css` might be better than the above in some ways (though arguably a little less "encapsulated").

in any case, global styles definitely seem more than fine for now 👍 especially since this stuff for `button` feels really genuinely global (rather than being convenient short-term but hard to maintain long-term, which I think can sometimes happen when abstracting to global styles too early).